Nos/Views/DiscoverView.swift

```diff
@@ -43,18 +43,24 @@ struct DiscoverView: View {
             }
             .padding(.horizontal)
             .toolbar {
-                ToolbarItem(placement: .navigationBarTrailing) {
-                    Button {
-                        columns += 1
-                    } label: {
-                        Image(systemName: "plus")
-                    }
-                }
-                ToolbarItem(placement: .navigationBarTrailing) {
-                    Button {
-                        columns = max(columns - 1, 1)
-                    } label: {
-                        Image(systemName: "minus")
+                ToolbarItem(placement: ToolbarItemPlacement.principal) {
```

Comment:
I had to use .principal here in order for the search bar to take up horizontal space, but it also adds vertical space.